lib/dor/models/admin_policy_object.rb

```diff
@@ -4,6 +4,7 @@ class AdminPolicyObject < ::ActiveFedora::Base
     include Governable
     include Editable
     include Describable
+    include Permissable
     include Processable
     include Versionable
```

Comment:
Permissable is included everywhere that Governable is included.  This is open for consideration and revision.  See additional includes below.